pkg/ineffassign/testdata/testdata.go

```diff
@@ -670,3 +712,15 @@ func _() {
 	}
 	_ = x
 }
+
+func returnErr() error {
+	return nil
+}
+
+func _() (err error) {
+	if 1 > 2 {
+		err = returnErr() // want "ineffectual assignment to err"
+	}
+	err = returnErr()
+	return
+}
```

Comment:
This test fails with your changes (but not without), right?  I think you forgot to delete the diagnostic comment.

Reply:
No, I had it wrong.  This should pass.